include/service/entities/plugin_service.js

```diff
@@ -538,6 +538,65 @@ module.exports = function PluginServiceModule(pb) {
      * existing settings for the plugin are deleted before the new settings are
      * persisted.
      *
+     * @method syncSettings
+     * @param details The details object to extract the settings from
+     * @param cb A callback that provides two parameters: cb(error, TRUE/FALSE).
+     * TRUE if the settings were successfully cleared and reloaded. FALSE if not.
+     */
+    PluginService.prototype.syncSettings = function(plugin, details, cb) {
+        var self = this;
+        this.getSettingsKV(plugin.dirName, function(err, settings) {
```

Comment:
Again, as mentioned below, the PluginService used here's site needs to be mapped to the plugin in question for this to work properly. Ideas to the best way to handle this other than instantiating a new PluginService each time?